op-deployer/pkg/deployer/state/deploy_config_test.go

```diff
@@ -34,7 +35,7 @@ func TestCombineDeployConfig(t *testing.T) {
 		},
 	}
 	state := State{
-		SuperchainDeployment: &SuperchainDeployment{ProtocolVersionsProxyAddress: common.HexToAddress("0x123")},
```

Comment:
The tiniest bit of info I found while refactoring: you can also do `common.Address{'A'}`